.gitlab/e2e/e2e.yml

```diff
@@ -750,7 +750,7 @@ new-e2e-otel-eks:
     - !reference [.needs_new_e2e_template]
     - qa_dca
     - qa_agent
-    - qa_agent_ot
+    - qa_agent_full
```

Comment:
This change alone looks suspicious to me.
I didn’t think that the e2e tests framework was able to check the Gitlab jobs dependencies to asses which images need to be tested.
So, with only this change, I would have expected the e2e tests to fail because of missing `-ot-beta` images.
If the e2e tests are not failing, then, it would mean that they’re not using the `-ot-beta`/`-full` images and this line can simply be deleted.
Or am I missing something subtle?

Reply:
[This](https://github.com/DataDog/test-infra-definitions/pull/1505) and the bump made [here](https://github.com/DataDog/datadog-agent/pull/36033) was already merged, so the tests should already be using the `full` images.